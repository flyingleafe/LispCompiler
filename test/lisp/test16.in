1234567890
_beforespaceis1