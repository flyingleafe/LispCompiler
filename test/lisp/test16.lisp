;;; Testing printStr/readStr and string-lisp conversions
(printString "This is the start of test")
(printString (readString))
(printString "This is the middle of test")
(printString (cons 49 (readString)))
(printString "This is the end of test")
