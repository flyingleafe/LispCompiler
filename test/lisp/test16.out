This is the start of test
1234567890
This is the middle of test
1_beforespaceis1
This is the end of test
